test/test_dists.jl

```diff
@@ -481,17 +481,17 @@ Q = Q * Q'  # make sure Q is positive-definite
 end #testset
 
 @testset "Euclidean precision" begin
-    X = [0.1 0.2; 0.3 0.4; -0.1 -0.1]
-    pd = pairwise(Euclidean(1e-12), X, X)
-    @test pd[1,1] == 0
-    @test pd[2,2] == 0
-    pd = pairwise(Euclidean(1e-12), X)
-    @test pd[1,1] == 0
-    @test pd[2,2] == 0
-    pd = pairwise(SqEuclidean(1e-12), X, X)
-    @test pd[1,1] == 0
-    @test pd[2,2] == 0
-    pd = pairwise(SqEuclidean(1e-12), X)
-    @test pd[1,1] == 0
-    @test pd[2,2] == 0
+X = [0.1 0.2; 0.3 0.4; -0.1 -0.1]
```

Comment:
Ah, all of the other `@testset`s were the other way. I've corrected them all now